es-vue-base/src/lib-components/EsHorizontalList.vue

```diff
@@ -28,7 +28,11 @@
             <es-button
                 v-if="arrows"
                 variant="link"
-                class="text-black d-none d-lg-block p-0 mr-200"
+                :disabled="atBeginning"
+                class="d-none d-lg-block p-0 mr-200"
```

Comment:
The design has the chevrons a bit closer to the circles. I doubt this design passes the tap-targets accessibility test as those circles and chevrons are pretty close together, but that part is out of scope - to better match the design without making the chevrons too close, maybe we bump this `mr-200` down to `mr-100`?

Reply:
Ah yes good catch!